inscriptions/ISic000696.xml

```diff
@@ -198,19 +198,23 @@
         <body>
             <div type="edition" xml:space="preserve" xml:lang="la" resp="#JH">
                 <ab>
-                    <lb n="1"/><name>Iulia</name> <name>Gamice</name>
-                    <lb n="2"/><expan><abbr>vix</abbr><ex>it</ex></expan> <expan><abbr>ann</abbr><ex>is</ex></expan> <num value="70">LXX</num> <expan><abbr>h</abbr><ex>ic</ex></expan> <expan><abbr>s</abbr><ex>ita</ex></expan> <expan><abbr>e</abbr><ex>st</ex></expan>
-                    <lb n="3"/><name>Hygia</name> <supplied reason="lost">fili</supplied>a
-                    <lb n="4"/>matri o<supplied reason="lost">pti</supplied>mae
+                    <lb n="0"/><gap reason="lost" extent="unknown" unit="line"/>
+                    <lb n="1"/><supplied reason="lost"><gap reason="lost" extent="unknown" unit="character"/><expan><abbr>proc</abbr><ex>uratori</ex></expan> f</supplied>a<unclear>m</unclear><supplied reason="lost"><expan><abbr>il</abbr><ex>iarum</ex></expan> <expan><abbr>glad</abbr><ex>iatoriarum</ex></expan></supplied>
```

Comment:
The `<gap reason="lost" extent="unknown" unit="character"/>` comes first and sits outside the supplied. For simplicity and to avoid problems, it makes sense, in turn, to create a separate supplied around each word. so:
`<gap reason="lost" extent="unknown" unit="character"/> <supplied reason="lost"><expan><abbr>proc</abbr><ex>uratori</ex></expan></supplied>` etc.

And for the next one, the expan for familiarium needs to embrace the whole word, with supplied inside where necessary (sorry, this is a messy one!) so:
`<expan><abbr><supplied reason="lost">f</supplied>a<unclear>m</unclear><supplied reason="lost">il</supplied></abbr><supplied reason="lost"><ex>iarum</ex></supplied></expan>`
 
And then all of gladiatoriarum can simply be placed inside another supplied.